src/main/java/ch/uzh/ifi/hase/soprafs24/service/SpotifyService.java

```diff
@@ -90,5 +97,40 @@ public static HashMap<String, String> getUserData(String accessToken) {
         }
 
         return spotifyUserData;
+    }
+
+    public static PlaylistCollectionDTO getUserPlaylistNames(String accessToken){
```

Comment:
We need to align this with our security construct (we do not pass the access token around)